### ISSUE TITLE
Beautify Zoom-error-dialog to be more accurate

### ISSUE THREAD

**Issue by esaruoho** [surge-synthesizer/surge#433]
"Your screen is 1920x1177 and your zoom of 200% would make your surge 1808x-1084".

Not sure what that is, or why I can't get 200% even though i can.

![screenshot_31_01_2019__16_43](https://user-images.githubusercontent.com/4966687/52061662-7d989b80-2577-11e9-8484-3071741b63d2.jpg)

**Comment by esaruoho** [comment]
surge capitalisation PR at #434 - the rest i'll leave up to @baconpaul

**Comment by baconpaul** [comment]
Oh window decoration eats about 5-10% of pixels so I cropped it below that. Although we should change the error message to be less annoying in that case agreed!

The error is pretty easy to find. Look for a prompterror in surgeguieditor.cpp setzoomfactor I think. Happy for you to rewrite it if you want!

**Comment by esaruoho** [comment]
yeah i wrote it.. but the weird -1084 throws me off.. why would be a height be in minus

**Comment by esaruoho** [comment]
Well, we can move some of the text here:

1440x877 cannot be resized higher than 135% - not even 145%.

**Comment by esaruoho** [comment]
Overall feels like Zoom could be a bit more dangerous, and maybe a bit more precise:

When resizing to 150%

`You attempted to resize Surge to a size larger than your screen. Your screen is 1440x877 and your zoom of 150% would make your Surge 1356x-813`

When resizing from 135% to 145%..

`You attempted to resize Surge to a size larger than your screen. Your screen is 1440x877 and your zoom of 145% would make your Surge 1310.8x-785.9 - Retaining current zoom of 135%.`

**Comment by esaruoho** [comment]
these are two different things, however, @baconpaul  - this was about tweaking the text to be accurate, the other issue was about not being able to get to "semi-large" screen sizes of surge without running into issues..

**Comment by baconpaul** [comment]
Yup but it's all the same .9 choice

**Comment by baconpaul** [comment]
#456 #457 #473 are the other related issues in this "zoom family"

**Comment by baconpaul** [comment]
I'm taking on of the "Size"s out of the title. So "Limiting Zoom by Screen Size"
